packages/epics/src/membership/components/outer-spaces/outer-spaces-list.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from 'react';
+import { OuterSpaceCard } from './outer-space-card';
+import { useOuterSpaces } from '../../hooks/use-outer-spaces';
+
+type OuterSpacesListProps = {
+  page: number;
+  activeSort: string;
+};
+
+const OuterSpacesList: FC<OuterSpacesListProps> = ({ page, activeSort }) => {
```

Comment:
Let's always use named exports. I know, default exports for components sticks like a bad disease in the react community. There is no good reason for it - just some extra line and possible confusion. Especially when mixed implementation styles (sometimes default sometimes named exports). 

```suggestion
export const OuterSpacesList: FC<OuterSpacesListProps> = ({ page, activeSort }) => {
```